maintainers/maintainer-list.nix

```diff
@@ -14045,6 +14045,12 @@
     githubId = 1449319;
     name = "Pacien Tran-Girard";
   };
+  pacificviking = {
+    email = "johnhaoallwood@gmail.com";
+    github = "PacificViking";
+    githubId = 32098177;
+    name = "John Hao";
+  };
```

Comment:
please split this into it's own commit called `maintainers: add pacificviking`

Reply:
Good point, thanks. (that was an accidental ping from the rebase sorry about that)